csun-metalab/nomi
# NAM-515 -- UI: Email in student profile overflows past the initial container in smaller screens

# Description
When you visit any students profile, the email will clip outside of the container if the person has a long name.
  
# Testing
1. Visit a students profile and ensure the email doesn't clip out of the container at different viewports.
2. Ensure the font scaling looks good and makes sense.
